vg/draw/canvas.go

```diff
@@ -33,20 +33,24 @@ type TextStyle struct {
 	// Font is the font description.
 	Font vg.Font
 
-	// Rotation is the text rotation in radians, counterclockwise from
-	// the default drawing direction.
+	// Rotation is the text rotation in radians,
+	// counterclockwise from the default drawing
+	// direction. (In English, the default drawing
```

Comment:
I don't like this here.

Discussion of this also feels like a package-level comment.